pkg/serverless/logs/logs.go

```diff
@@ -275,8 +280,14 @@ func processLogMessages(c *LambdaLogsCollector, messages []logMessage) {
 				continue
 			}
 			ecs := c.ExecutionContext.GetCurrentState()
-			logMessage := logConfig.NewChannelMessageFromLambda([]byte(message.stringRecord), message.time, ecs.ARN, ecs.LastRequestID)
-			c.LogChannel <- logMessage
+			if len(message.objectRecord.requestID) > 0 {
```

Comment:
```suggestion
			if message.objectRecord.requestID != "" {
```

This is a nit pick. Comparing to an empty string will be more performant.